File: testers/avro/java-vanilla/generated-and-checked-in/com/example/events/consumer/LinkedListNodeHandler.java
package com.example.events.consumer;

import com.example.events.LinkedListNode;
import com.example.events.header.StandardHeaders;

/** Handler interface for linked-list-node topic events */
public interface LinkedListNodeHandler {
  /** Handle a message from the topic */
  void handle(String key, LinkedListNode value, StandardHeaders headers);
}

File: testers/avro/scala/generated-and-checked-in/com/example/events/consumer/LinkedListNodeHandler.scala
package com.example.events.consumer

import com.example.events.LinkedListNode
import com.example.events.header.StandardHeaders

/** Handler interface for linked-list-node topic events */
trait LinkedListNodeHandler {
  /** Handle a message from the topic */
  def handle(
    key: String,
    value: LinkedListNode,
    headers: StandardHeaders
  ): Unit
}